src/triton_cli/server/server_factory.py

```diff
@@ -200,3 +184,23 @@ def _get_triton_server_config(config):
                 triton_config["log-verbose"] = "1"
 
         return triton_config
+
+    @staticmethod
+    def _get_openai_chat_template_tokenizer(config):
+        """
+        Raises an exception if a tokenizer can not be found and is not specified with OpenAI Frontend
+        """
+        if config.openai_chat_template_tokenizer:
+            return config.openai_chat_template_tokenizer
+
+        trtllm_utils = TRTLLMUtils(config.model_repository)
+        vllm_utils = VLLMUtils(config.model_repository)
+
+        if trtllm_utils.has_trtllm_model():
+            return trtllm_utils.get_engine_path()
+        elif vllm_utils.has_vllm_model():
+            return vllm_utils.get_vllm_model_huggingface_id_or_path()
```

Comment:
Can you add a log info message with the detected tokenizer to give some insight to the user?